README.md

```diff
@@ -111,16 +106,13 @@ Users that contributed between December 1, 2020, 12:00 AM GMT-6 and December 15,
 
 For further reading see the [luxon time zone documentation](https://moment.github.io/luxon/docs/manual/zones) and the [List of tz values](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
 
--   The CSV you input will be turned into an array, so the numbers for the CSV columns are zero-indexed. For example, in the example CSV above, `CSV_COLUMN_NUMBER_FOR_GITHUB_ID = 0` and `CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID = 1`. If you choose not to use an alternate id, you can put the same column number in both `CSV_COLUMN_NUMBER_FOR_GITHUB_ID` and `CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID`.
-
--   To filter out events for which the author is the owner of the repository, simply set the `IGNORE_SELFOWNED_EVENTS` environment variable to "true", otherwise leave as `IGNORE_SELFOWNED_EVENTS = ""`.
-
 ### To run:
 
 In your terminal, type `cat {path/to/CSVfile}.csv | node index.js {date1} {date2}`
 
 > In the above, any text inside of curly brackets {} means that you should put your own value in.  
-> Dates should be written in ISO-8601 format. For example, April 1, 2019 should be entered as 2019-04-01.
+> Dates should be written in ISO-8601 format. For example, April 1, 2019 should be entered as 2019-04-01.  
+> Here's an example of what the terminal command could look like (I name my CSV files by date): `cat CSVsToParse/2020-11-01.csv | node index.js 2020-10-01 2020-11-01`
 
 Reminder: You can pipe the output to a file, if you like: `cat {path/to/CSVfile} | node index.js > {nameOfFileToCreate}.txt {date1} {date2}`
```

Comment:
The other day, I noticed that we use the word "pipe" here, which is arguably correct, but is easy to confuse with the pipe symbol (|). I would suggest changing this to "You can redirect the output to a file...". 

If we switch the recommended command above, this would become `node index.js {date1} {date2} < {path/to/CSVfile} > {nameOfFileToCreate}.txt `

Reply:
Good point! Thanks.